OpenMEEG/cmake/FindMKL.cmake

```diff
@@ -5,7 +5,9 @@
 #  MKL_ROOT_DIR - path to the MKL base directory
 #  MKL_INCLUDE_DIR - the MKL include directory
 #  MKL_LIBRARIES - MKL libraries
+#  MKL_LIBRARY_DIR - MKL library dir (for dlls!)
 #
+# we use mkl_link_tool to get the library needed depending on variables
 # There are few sets of libraries:
```

Comment:
"we use mkl_link_tool to get the library needed depending on variables"
instead of selecting by ourself the libs